set-client/src/main/java/eu/europeana/set/client/UserSetApiClient.java

```diff
@@ -34,34 +36,50 @@ public SearchUserSetApi getSearchUserSetApi() {
         return searchUserSetClient;
     }
 
+    /**
+     * Authentication token for the Translation api requests.
+     * @param authToken
+     */
+    public void setAuthToken(String authToken) {
+        token.set(authToken);
+    }
+
+    /**
+     * Close / purge the token from memory
+     */
+    public void close() {
```

Comment:
close method is typically used for closing streams, we should rename this method to somthing else, somthing like "logout",